docker/test/stress/run.sh

```diff
@@ -376,7 +376,7 @@ else
 fi
 
 # Put logs into /test_output/
-for log_file in /var/log/clickhouse-server/clickhouse-server.log*
+for log_file in /var/log/clickhouse-server/clickhouse-server*.log
```

Comment:
It's interesting that all logs were successfully added to the report despite the fact that `run.sh` did not put them into `test_output`. It's because we have a kind of code duplication in `tests/ci/stress_check.py`. Maybe we can remove some code?

Reply:
All logs were added to the report but they weren't compressed. As I understand, these lines in run.sh are needed just to compress all logs to use less space.

Reply:
Some of them were and some were not: 
https://s3.amazonaws.com/clickhouse-test-reports/36200/99ea71b3717b3c078f684abfac1f78562f1aa6d8/stress_test__thread__actions_.html

Probably it's because all large files are compressed by default

Reply:
> Probably it's because all large files are compressed by default

Yep, found this place:
https://github.com/ClickHouse/ClickHouse/blob/71fb04ea4ad17432baba7934a10217f5f6a1bde3/tests/ci/s3_helper.py#L45

Then I think we can remove this code from run.sh

Reply:
I saw these lines, but decided to leave compression in stress tests explicitly, since otherwise it will compress logs only if it's size is 64MB, which I found quite huge.

To me it makes sense to keep this limit up to say, 1mb (or maybe even less), since everything that is bigger you cannot open in the browser anyway and you need to download it anyway.

So, what do you think about decreasing this limit to, say, 1mb?